### DISCUSSION TITLE
Installation method without jsonnet-bundler

### DISCUSSION THREAD

**Discussion by jacksongoode** [grafana/grafonnet#95] [Q&A]
It would be great to also release a zip package of the library for those who just want to download, extract and import.

**Comment by Duologic** [comment]
Sorry, I've left this hanging for a while as I didn't have a proper response.

With jsonnet-bundler being the defacto standard package manager, we are only supporting jsonnet-bundler as an install method. Additionally to installing Grafonnet it also imports external dependencies, packaging this up manually would raise quite a few questions about dependencies and versioning.

I suppose you can set up a simple process yourself that runs jsonnet-bundler to collect all packages and zips them up with the versions that you wish to use. Or alternatively, do a git checkout manually (which is what jsonnet-bundler does internally anyway).

**Comment by alias-dev** [comment]
[git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) are another option for using this in another repo without using the bundler
